backend/src/Entity/Document.php

```diff
@@ -36,6 +30,14 @@ class Document extends Node
     #[ORM\Column]
     private ?bool $under_review = null;
 
+    private ?string $contentUrl = null;
```

Comment:
should be content_url to be consistent

Reply:
Is content url nodig? getters en setters worden nergens gebruikt, en wordt ook niet opgeslagen in database. indien nodig, mss comment bij zetten waarom
Ben niet zeker maar als dit dient voor de frontend moet in principe enkel DocumentApi de content url opslaan en Document zelf niet

Reply:
removed contentUrl, was sws niet opgeslagen in db